tests/ApplicationTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Console\Tests;
+
+use Psr\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\Console\Tester\CommandTester;
+use Yiisoft\Yii\Console\ExitCode;
+use Yiisoft\Yii\Console\Event\ApplicationStartup;
+use Yiisoft\Yii\Console\Event\ApplicationShutdown;
+
+final class ApplicationTest extends TestCase
+{
+    public function testGetDispatcher(): void
```

Comment:
No need for this test since return type is already specified explicitly:

```php
public function getDispatcher(): EventDispatcherInterface
{
    return $this->dispatcher;
}
```

Reply:
done.